sdk/attestation/azure-security-attestation/tests/test_attestation.py

```diff
@@ -162,6 +164,7 @@ def test_aad_getopenidmetadata(self, attestation_aad_url):
 
     @pytest.mark.live_test_only
     @AttestationPreparer()
+    @recorded_by_proxy
```

Comment:
THis is a live-only test, should it have the recorded_by_proxy preparer attached?

Reply:
Without this, the tests result into error while running in live mode due to fixture in conftest
Something like below:
**fixture 'attestation_aad_url' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, clean_cached_resources, cov, doctest_namespace, environment_variables, event_loop, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recorded_test, recwarn, start_proxy, test_proxy, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, variable_recorder
>       use 'pytest --fixtures [testpath]' for help on them.**

Is there any other way to mitigate this without adding @recorded_by_proxy annotation? Meanwhile I will try to understand if there is any other way possible